src/views/VenuesListWithFilter/FilterList/PriceRange/index.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { StyledSlider } from '../../../../shared/components/StyledSlider/index'
+
+export function PriceRange ({ priceValue, setPriceValue }) {
```

Comment:
```suggestion
export function PriceRange ({ priceValue, setPrice }) {
```
Usually the `value` suffix is redundant, it is self-explanatory that you are setting a value.